src/Client.js

```diff
@@ -7,12 +7,12 @@ export default class Client {
    * @param {String} options.baseURL Base URL of your ESN server
    * @param {String} options.auth    Auth stretagy
    */
-  constructor({ baseURL } = {}) {
+  constructor({ baseURL, auth } = {}) {
```

Comment:
Should we have some validators for auth too?

Reply:
Should we have tests to ensure this will not be missing ever again?

Reply:
> Should we have some validators for auth too?

IMO, no need for now. DEV uses the library and if auth is wrong, 401 response will be appear

Reply:
Added test